### DISCUSSION TITLE
Using Mount only for entirely independent apps?

### DISCUSSION THREAD

**Discussion by ni-aas** [encode/starlette#1964] [Q&A] [answered]
Can `Mount` be used to separate similar routes/views to better structure the project, even if it's not entirely independent and separate applications , or is it normal to do so? 

I have a "main" routes.py that mount together all the "folders" (for example "admin" and "auth"), and inside those there is a new folder (not entirely separate app) with it's own routes/views, and so on. Usually just separated based on "/admin/_users_/nnn" where "users" get's it's own folder. In the template that url would be `'admin:users:nnn'`.

For learning purposes I would like to know what you think?
Thanks for your time and help.

**Comment by alex-oleshkevich** [comment, marked as answer]
Yes, that's totally fine. Use Mount to mount apps or child routes:
```python
routes = [
	Route('/', homepage),
	Mount('/app-prefix', app=AnotherASGIApp)
	Mount('/prefix', routes=[
		Route('/', view),
		Route('/another-one', view),
		Mount('/level-2', routes=[...]),
	])
]
```
